README.md

```diff
@@ -10,17 +10,19 @@ This repo contains 2 jvm packages:
 
 # Usage
 
-tbdex is published to maven central but some additional repositories are needed for transitive dependencies currently: 
+tbdex sdk is consumable through Maven Central but some additional repositories are needed for transitive dependencies currently: :
 
 ## Gradle
-
 ```kotlin
 repositories {
+  mavenCentral()
   maven("https://jitpack.io")
   maven("https://repo.danubetech.com/repository/maven-public/")
 }
 
 dependencies {
+  implementation("com.github.TBD54566975:tbdex:0.4.0-beta")
```

Comment:
Do we need to update this to reflect the new group id and version?

Reply:
oh yea. i missed this in merge conflict fix. ty!